packages/components/Button/src/FAB/__snapshots__/FAB.test.tsx.snap

```diff
@@ -116,9 +114,8 @@ exports[`Default FAB (iOS) 1`] = `
       Object {
         "alignItems": "center",
         "alignSelf": "flex-start",
-        "backgroundColor": "#0078d4",
+        "backgroundColor": undefined,
```

Comment:
Is this an expected change - would it impact any visuals in non iOS components..?

Reply:
Yes it's expected, no should not impact any of the non-iOS platforms. Might get updated if I can bump to 0.31.0 instead of 0.29.0

Reply:
Updated to 0.31.0, there should be no additional undefined values now